website_hr_department/__openerp__.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#     This file is part of website_hr_department, an Odoo module.
+#
+#     Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+#
+#     website_hr_department is free software: you can redistribute it and/or
+#     modify it under the terms of the GNU Affero General Public License
+#     as published by the Free Software Foundation, either version 3 of
+#     the License, or (at your option) any later version.
+#
+#     website_hr_department is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU Affero General Public License for more details.
+#
+#     You should have received a copy of the
+#     GNU Affero General Public License
+#     along with website_hr_department.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': "Departments Page",
+    'summary': """
+        Display the structure of your departments and their members.
```

Comment:
yes it appears well
![image](https://cloud.githubusercontent.com/assets/544090/6391461/adfaf28c-bdb6-11e4-85bc-c79197caf40a.png)
![image](https://cloud.githubusercontent.com/assets/544090/6391484/d65f9c32-bdb6-11e4-9a9b-b3d2baf8ec5d.png)